codalab/rest/interpret.py

```diff
@@ -173,10 +193,20 @@ def fetch_interpreted_worksheet(uuid):
     # Fetch items.
     worksheet_info['raw'] = get_worksheet_lines(worksheet_info)
 
-    # Replace searches with raw items.
+    # Expand user-written searches (including bundle & worksheet) to raw items.
     # This needs to be done before get_worksheet_lines because this replaces
     # user-written raw items.
-    worksheet_info['items'] = expand_raw_items(worksheet_info['items'])
+    # Also stores the mapping from raw items to the source line that produced them.
+    # This allows us to manipulate the source lines easier on the frontend.
+    expanded_items_to_raw_lines = []
```

Comment:
Can we be consistent with naming? either call both raw_item_to_source_line or expanded_items_to_raw_lines?